src/cobalt.rs

```diff
@@ -176,12 +176,15 @@ pub fn build(config: &Config) -> Result<()> {
         .collect();
 
     trace!("Generating other documents");
+    let timestamp = Value::Str(UTC::now().timestamp().to_string());
```

Comment:
> I used Value::Str since Value::Num only accepts f32 - the timestamp is i64 so casting to f32 trims the number to the point where it's not unique.

liquid-rust/#88 will require us to have integer values.  So in the future we can change this.